### ISSUE TITLE
Wrong MSW ctx.text type is applied

### ISSUE THREAD

**Issue by zerocity** [anymaniax/orval#272]
In msw mock the ctx.json / ctx.text can be wrong set
https://github.com/anymaniax/orval/blob/a443ee4d73e338bef88b7b31620d013c9655a4fe/src/core/generators/msw.ts#L95

example json responses what are not typed as "application/json"

```
   "application/ld+json": { "schema": { "$ref": "#/components/schemas/business.jsonld" } },
    "application/hal+json": { "schema": { "$ref": "#/components/schemas/business" } },
    "application/vnd.api+json": { "schema": { "$ref": "#/components/schemas/business" } },
    "application/json": { "schema": { "$ref": "#/components/schemas/business" } },
    "text/html": { "schema": { "$ref": "#/components/schemas/business" } }
```

related to issue : https://github.com/anymaniax/orval/issues/232

**Comment by anymaniax** [comment]
Hello @zerocity, Thanks!  Would be better to do it the other way around and also maybe handle more cases. Do you want to do the modification yourself?

**Comment by anymaniax** [comment]
fixed with `6.4.1`
